anonymity.md

```diff
@@ -0,0 +1,52 @@
+# Anonymity on [Mapknitter](https://mapknitter.org)
+
+[Publiclab](https://publiclab.org) is an organization dedicated to protecting the planet by building useful [tools](https://github.com/publiclab) and bringing together passionate and kind people who want to have a positive impact. Mapknitter was at the beginning of this collaboration between committed individuals when it was used to [provide public information about the 2010 BP Oil Disaster](https://publiclab.org/wiki/stories). It has since grown and now hosts **2761** maps yet it still upholds its fundamental idea of sharing **open** data that anyone can **contribute to or edit**. This mission is why the ability to contribute anonymously is important for this project. 
```

Comment:
Hi @Uzay-G thanks so much for opening this...Be patient with this one because its critical and may involve a few people so there might be some back and forth..here are my some suggestions.
I think we should say `more than 2761 maps` because the number should keep increasing.
Also  I would suggest this part `This mission is why the ability to contribute anonymously is important for this project` 
to not follow the previous line because it looks like its a continuation of the previous line that has this phrase `data that anyone can contribute to or edit` because the anymous authors cannot edit, delete  or comment on other maps...Does this make sense?

Reply:
Yeah ok no problem :smile:. Do you think it's a good start? I will implement your changes now. Thanks for reviewing it